### DISCUSSION TITLE
Feature request: Toggle to allow IR codes to be sent when no stream active.

### DISCUSSION THREAD

**Discussion by blasco403** [GerryDazoo/Slinger#547] [General]
Hey Gerry, wondering if in a future version there could be a toggle to allow for web remote to send IR codes when no stream active.

It would be nice if it could be per configured slingbox (one could ignore IR and another would accept).

I would like the option to use the nice web remote I created on my phone to control slingbox when watch locally on my TV.

Thanks.

**Comment by GerryDazoo** [comment]
to send the remote codes you need a control session open to the slingbox. It's doable but it would require a complete re-factoring of the current code. So don't expect to see it anytime soon.
I'm a little confused. You want to be able to use your web remote for the slingbox to control your STB while watching the TV locally and not streaming? You must really like your custom remote!

What about a button on your fancy remote that would start the stream in background and then just toss the returned data.

**Reply by blasco403** [reply]
Understood, sounds like it's not worth all your effort!!

And yes I like my web remote.... Single button for fav channels and a scrollable channel list - would be easier then using the actual STB remote when local.
